### PR TITLE
New package: FastFetchCLI.FastFetch version 2.13.2

### LINKED ISSUE #154801: [Package Request]: fastfetch
### How can we help?

None

### Please read and ensure the following

- [X] The installer meets the above requirements

### Please provide the following information

Download Page Url: https://github.com/fastfetch-cli/fastfetch/releases
Publisher: fastfetch-cli
Package Name: fastfetch
Description: Like neofetch, but much faster because written mostly in C.
Package Version: 2.13.2
Installer URL: https://github.com/fastfetch-cli/fastfetch/releases/download/2.13.2/fastfetch-windows-amd64.zip

### PR DESCRIPTION
**Pull request by schembriaiden**
### Pull request has been created with [komac](https://github.com/russellbanks/Komac) v2.2.1 :rocket:

Resolves: https://github.com/microsoft/winget-pkgs/issues/154801
 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/154805)

### CODE REVIEW

**Comment by vikingnope** [comment]
I tested this locally and it worked fine for me

**Comment by stephengillie** [comment]
Hi @vikingnope,

The package didn't pass a Defender or similar security scan. This might be a false positive and we can rescan tomorrow.

(Automated response - build 881.)

**Comment by stephengillie** [comment]
Automatic Validation ended with:

> Installation failed with exit code -1978335187
> 2024-05-23 01:03:09.594 [FAIL] Installer failed security check. Url: https://github.com/fastfetch-cli/fastfetch/releases/download/2.13.2/fastfetch-windows-i686.zip Result: 0x80004005

Automated error analysis suggests -1978335187 may mean Installer failed security check. Another script should retry this every 18 hours. 

(Automated response - build 881.)

File: manifests/f/FastFetchCLI/FastFetch/2.13.2/FastFetchCLI.FastFetch.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Created with komac v2.2.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: FastFetchCLI.FastFetch
```
Comment (stephengillie) [inline manifests/f/FastFetchCLI/FastFetch/2.13.2/FastFetchCLI.FastFetch.locale.en-US.yaml]:
```suggestion
PackageIdentifier: Fastfetch-cli.Fastfetch
```
Please update file name and path to match this change.

**Review by stephengillie — CHANGES_REQUESTED** [review]
Hi @vikingnope, 

This package appears to have been recently added under a slightly different `PackageIdentifier`:

- https://github.com/microsoft/winget-pkgs/pull/154718

Should this package's `PackageIdentifier` be modified to match?

**Pull request closed without merging**